lib/rabbitmq/cli/ctl/commands/list_unresponsive_queues_command.ex

```diff
@@ -25,35 +27,50 @@ defmodule RabbitMQ.CLI.Ctl.Commands.ListUnresponsiveQueuesCommand do
 
   def formatter(), do: RabbitMQ.CLI.Formatters.Table
 
+  @info_keys ~w(name durable auto_delete
+            arguments pid recoverable_slaves)a
+
+  def info_keys(), do: @info_keys
+
   def scopes(), do: [:ctl, :diagnostics]
 
-  def validate(_, _), do: :ok
+  def validate(args, _opts) do
+      case InfoKeys.validate_info_keys(args, @info_keys) do
+        {:ok, _} -> :ok
+        err -> err
+      end
+  end
 
-  def merge_defaults(args, opts) do
+  def merge_defaults([_|_] = args, opts) do
     {args, Map.merge(default_opts(), opts)}
   end
+  def merge_defaults([], opts) do
+      merge_defaults(~w(name), opts)
+  end
 
   def switches(), do: [queue_timeout: :integer, local: :boolean]
 
   def usage() do
-    "list_unresponsive_queues [--local] [--queue-timeout <queue-timeout>]"
+    "list_unresponsive_queues [--local] [--queue-timeout <queue-timeout>] [<queueinfoitem> ...]"
```

Comment:
Info items for this command are not the same as `queueinfoitem` for the `list_queues` command and they are validated (so not all `queueinfoitem` can be used with this command). When printing `rabbitmqctl help` this can be confusing.